mm-commotion-mac2ip.tests

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# � 2016 Meta Mesh Wireless Communities. All rights reserved.
```

Comment:
Something happened to the copyright symbol here. NBD, fix it only if there are other things to fix.